webknossos/webknossos/cli/_utils.py

```diff
@@ -17,7 +17,7 @@
 class DistributionStrategy(str, Enum):
     """Enum of available distribution strategies.
 
-    TODO  pylint: disable=fixme
+    TODO
```

Comment:
it might make sense to add https://docs.astral.sh/ruff/rules/line-contains-todo/. this would catch unintentional todo comments in PRs before merging.

Reply:
Done